packages/config/src/android/Manifest.ts

```diff
@@ -173,21 +173,23 @@ export function getMainActivity(manifest: Document): ManifestActivity | null {
 export function addMetaDataItemToMainApplication(
   mainApplication: any,
   itemName: string,
-  itemValue: string
+  itemValue: string,
+  itemType: 'resource' | 'value' = 'value'
```

Comment:
wasn't sure whether or not to make this an `options` object instead, to allow for more customization down the line, but `resource` and `value` are the only two options: https://developer.android.com/guide/topics/manifest/meta-data-element